src/test-utils.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MockedProvider, MockedResponse } from '@apollo/client/testing';
+// this adds custom jest matchers from jest-dom
+import '@testing-library/jest-dom/extend-expect';
+
+type RenderApolloOptions = {
+  mocks?: MockedResponse[];
+  addTypename?: any;
+  defaultOptions?: any;
+  cache?: any;
+  resolvers?: any;
+  [st: string]: any;
+};
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+const renderApollo = (
+  node: any,
```

Comment:
Codacy found an issue: [Argument 'node' should be typed with a non-any type.](https://app.codacy.com/gh/Wyvarn/space-xplorer-client/pullRequest?prid=6087079)